main.js

```
@@ -481,6 +494,595 @@ function applyDanmakuFilterDensity(density) {
+
+// 简繁转换器(OpenCC hk→cn,与 overlay/lib/opencc.min.js 同一库同参数)。
+// opencc.min.js 是 UMD bundle,无 DOM 依赖,file.read + eval 即可在 JavaScriptCore 运行;
+// 失败则返回 false(列表退化为原文,不影响渲染主链路)。首次构建 ~几百 ms,缓存复用。
+function ensureBrowserSimplifier() {
+  if (openccSimplifier) return true;
+  if (!danmakuForceSimplified) return false;
+  if (!pluginRootPath) return false;
+  try {
+    var code = file.read(pluginRootPath + '/overlay/lib/opencc.min.js');
+    if (!code) return false;
+    eval(code); // UMD 挂到全局(globalThis.OpenCC)
```

Biome (2.5.6)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)